libs/core/langchain_core/prompts/loading.py

```diff
@@ -43,7 +43,7 @@ def _load_template(var_name: str, config: dict) -> dict:
         template_path = Path(config.pop(f"{var_name}_path"))
         # Load the template.
         if template_path.suffix == ".txt":
-            with open(template_path) as f:
+            with open(template_path, "r", encoding="utf-8") as f:
```

Comment:
Not sure this is foolproof - might need to add error handling for files that are not readable by utf-8 encoding

Reply:
@baskaryan I don't think this change is breaking, but I think it could be dealt with better, lmk what you think